Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.h

```diff
@@ -32,6 +32,17 @@ class QWheelEvent;
 
 namespace AzToolsFramework
 {
+    enum class CursorInputMode {
+        CursorModeNone,
+        CursorModeCaptured,   //!< Sets whether or not the cursor should be constrained to the source widget and invisible.
+                              //!< Internally, this will reset the cursor position after each move event to ensure movement
+                              //!< events don't allow the cursor to escape. This can be used for typical camera controls
+                              //!< like a dolly or rotation, where mouse movement is important but cursor location is not.
+        CursorModeWrapped,    //!< Flags whether the curser is going to wrap around the soruce widget.
+        CursorModeWrappedX,
+        CursorModeWrappedY 
```

Comment:
nit: It might be worth adding API comments for each enumeration just for completeness